features/other_a/page_title.feature

```diff
@@ -31,3 +31,13 @@ Scenario: Page title should change when tags are edited
   When I view the work "New Story"
   Then the page title should include "Harry Potter"
   And the page title should not include "Stargate"
+
+Scenario: Page title should be informative on the adult content notice page
+
+  Given I am logged in as "author"
+  And I post the 2 chapter work "New Story" with fandom "Stargate" with rating "Mature"
```

Comment:
Usually we indent `And` and `But` by one step. Not sure why the rest of the file doesn't do this, it might be older than that style guideline.